WPFMainApp/MainWindow.xaml.cs

```diff
@@ -62,16 +61,21 @@ private void AdjustInitPosition()
 
     private Point GetMousePosition()
     {
-
         var mousePosition = new Interop.Win32Point();
         Interop.GetCursorPos(ref mousePosition);
         var mousePositionPoint = new Point(mousePosition.X, mousePosition.Y);
         var actualMousePosition = ConvertCords(mousePositionPoint, this);
         return actualMousePosition;
     }
 
-    private void InitButtons(Browser[] browsers, double maxWidth, double maxHeight)
+    private void InitControls(Browser[] browsers, double maxWidth, double maxHeight)
     {
+        DefaultBrowserCheckBox.Content = string.Format(
+            "Always open links from \"{0}\" in selected browser", 
+            App.ParentProcess?.ProcessName ?? ""
+            );
+        DefaultBrowserCheckBox.Margin = new Thickness(2.0, 2.0, 2.0, 2.0);
```

Comment:
As far as margins are constantly here, plz move it to layout.

Reply:
Resolved in the recent commit